comprehensive/lms/templates/static_templates/404.html

```diff
@@ -5,5 +5,5 @@
 
 <section class="outside-app">
 <h1>${_("Page not found")}</h1>
-<p>${_('The page that you were looking for was not found. Go back to the {link_start}homepage{link_end} or let us know about any pages that may have been moved by contacting support.')}</p>
+<p>${_('The page that you were looking for was not found. Please contact <a href="/faq">support</a> and let us know.')}</p>
```

Comment:
Why put this on the user to report it? Doesn't analytics data provide this?

Reply:
I think it is basically a way to help the learner to point his complaint around the unreachable link - rather than the user doing our job to find out what is broken what is not.